bin/scripts/lib/fonts.json

```diff
@@ -70,6 +70,20 @@
       "repoRelease": true,
       "description": "Metrically similar to Arial, pan-European WGL character set, sans serif"
     },
+    {
+      "unpatchedName": "Atkinson Hyperlegible Mono",
+      "licenseId": "OFL-1.1-RFN",
+      "RFN": true,
+      "version": "1.0",
+      "patchedName": "AtkynsonMono",
+      "folderName": "AtkinsonMono",
+      "imagePreviewFont": "AtkynsonMono Nerd Font",
+      "imagePreviewFontSource": "AtkinsonHyperlegibleMono/AtkinsonHyperlegibleMono-Regular.ttf",
+      "linkPreviewFont": "atkinson-mono",
```

Comment:
This is wrong? At least it seems to be "atkinson-hyperlegible"; where did you get this from?